config/environment.rb

```diff
@@ -1,5 +1,7 @@
 # Load the Rails application.
 require_relative "application"
 
+ENV['CANVAS_URL'] ||= 'https://ucberkeley.test.instructure.com'
```

Comment:
don't put this in `environment.rb` as this file is not supposed to be installation-specific.  just document the fact that `CANVAS_URL` has to be set when using this adapter.

Reply:
Resolved in:
- [1d8ad98](https://github.com/cs169/flextensions/pull/32/commits/1d8ad9812304d313cc0eb683caa37b4934a65787)
- [979824f](https://github.com/cs169/flextensions/pull/32/commits/979824fc5dc03d630254a51b80d2200bce95c714)